aws/data_source_aws_glue_connection_test.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/glue"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDataSourceAwsGlueConnection_basic(t *testing.T) {
+	resourceName := "aws_glue_connection.test"
+	datasourceName := "data.aws_glue_connection.test"
+	rName := fmt.Sprintf("tf-testacc-glue-connection-%s", acctest.RandString(13))
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:   func() { testAccPreCheck(t) },
+		ErrorCheck: testAccErrorCheck(t, glue.EndpointsID),
+		Providers:  testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsGlueConnectionConfig(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccDataSourceAwsGlueConnectionCheck(datasourceName),
+					resource.TestCheckResourceAttrPair(datasourceName, "catalog_id", resourceName, "catalog_id"),
+					resource.TestCheckResourceAttrPair(datasourceName, "creation_time", resourceName, "creation_time"),
+					resource.TestCheckResourceAttrPair(datasourceName, "connection_type", resourceName, "connection_type"),
+					resource.TestCheckResourceAttrPair(datasourceName, "name", resourceName, "name"),
+				),
```

Comment:
lets also add `connection_properties` check here as we added it.

Reply:
I don't have `connection_properties` as part of the schema in the data source. It doesn't appear for every type of connection so I thought it might be best to leave it out.